spanner/spanner_snippets/spanner/spanner_create_database_with_property_graph.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2024 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spanner
+
+// [START spanner_create_database_with_property_graph]
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"regexp"
+
+	database "cloud.google.com/go/spanner/admin/database/apiv1"
+	adminpb "google.golang.org/genproto/googleapis/spanner/admin/database/v1"
+)
+
+func makeCreateDatabaseWithPropertyGraphRequest(instance string, dbName string) *adminpb.CreateDatabaseRequest {
```

Comment:
nit: I would combine this function and the next, so as to keep this code as copy-paste-able as possible.

Reply:
Done. Hopefully I understood your suggestion correctly.